music21/variant.py

```diff
@@ -947,12 +947,7 @@ def addVariant(
             tempVariant.append(sVariant)
         else:  # sVariant is not a measure
             sVariantMeasures = sVariant.getElementsByClass('Measure')
-            # apparently expression cannot be simplified. -- this is a mistake
-            # since sVariantMeasures will never == [] even if there are no measures.
-            # yet switching this to `if not sVariantMeasures` breaks things.
-            # TODO(msc) -- figure this out and fix it.
-            # noinspection PySimplifyBooleanCheck
-            if sVariantMeasures == []:  # If there are no measures, work element-wise
+            if not sVariantMeasures:  # If there are no measures, work element-wise
```

Comment:
Obviously we can back this out if you don't want to review this right now (should have been on a separate commit), but just noticed while fixing the variant test.

Reply:
It looks good -- nothing we have in the tests seems to be bitter by it.  I think it's time to bit the bullet and remove it, and see if bugs arise from it.